packages/react-accordion/src/components/AccordionHeader/useAccordionHeaderStyles.ts

```diff
@@ -22,10 +24,10 @@ const useStyles = makeStyles({
   root: theme => ({
     color: theme.colorNeutralForeground1,
     backgroundColor: theme.colorNeutralBackground1,
-    borderRadius: '2px',
+    ...shorthands.borderRadius('2px'),
   }),
   rootDisabled: theme => ({
-    backgroundColor: 'none',
+    backgroundImage: 'none',
```

Comment:
was this necessary ?
```suggestion
    backgroundColor: 'none',
```

Reply:
Not really 😉

⬇️⬇️⬇️

"none" is not a valid value for `background-color`:
https://developer.mozilla.org/en-US/docs/Web/CSS/background-color

![image](https://user-images.githubusercontent.com/14183168/143578758-975307f3-26cf-4da1-8c1d-8dab6a06b347.png)
https://developer.mozilla.org/en-US/docs/Web/CSS/background#formal_syntax

---

Or it's either `backgroundImage: 'none'` or `backgroundColor: 'initial'`

Reply:
As VR tests are passing, I am leaving it as it is. We can fix it if it will cause any problems.